guides.md

```diff
@@ -146,19 +146,28 @@ through the process of creating your own large-scale job submission, step-by-ste
 through the process of using the cluster, step-by-step.</p> -->
 
 <h2>Guides by Topic</h2>
-
-
-<br>
-<div class="card border-secondary" style="width: 31.5%;" >    
-	<img alt="Card image cap" class="card-img-top img-responsive" src="guide-icons/servers.png" style="margin-left: auto;margin-right: auto;width: 30%;"/>
-	<h5 class="card-title text-center">All Cluster Guides</h5>
-	<ul class="list-group list-group-flush" style="height: auto !important;">
-		<a href="/connecting.shtml"><li class="list-group-item list-group-item-action list-group-item-dark h-100" style="width: auto; height: auto !important; border-radius: 1px;">Connecting to CHTC</li></a>
-		<a href="/HPCuseguide.shtml"><li class="list-group-item list-group-item-action list-group-item-dark h-100" style="width: auto; height: auto !important; border-radius: 1px;">HPC Cluster Basic Use Guide</li></a>
-		<a href="/MPIuseguide.shtml"><li class="list-group-item list-group-item-action list-group-item-dark h-100" style="width: auto; height: auto !important;">MPI Use Guide</li></a>
-	</ul>
-</div>	
-
+<div class="card-deck">
+	<div class="card border-secondary h-100" >    
+		 <img alt="Card image cap" class="card-img-top img-responsive" src="guide-icons/servers.png" style="margin-left: auto;margin-right: auto;width: 30%;"/>
+		<h5 class="card-title text-center">HPC Cluster Guides</h5>
+		<ul class="list-group list-group-flush" style="height: auto !important;">
+			<a href="/connecting.shtml"><li class="list-group-item list-group-item-action list-group-item-dark h-100" style="width: auto; height: auto !important; border-radius: 1px;">Connecting to CHTC</li></a>
+			<a href="/HPCuseguide.shtml"><li class="list-group-item list-group-item-action list-group-item-dark h-100" style="width: auto; height: auto !important; border-radius: 1px;">HPC Cluster Basic Use Guide</li></a>
+			<a href="/MPIuseguide.shtml"><li class="list-group-item list-group-item-action list-group-item-dark h-100" style="width: auto; height: auto !important;">MPI Use Guide</li></a>
+		</ul>
+	</div>
+			
+	<div class="card border-secondary h-100" >    
+		 <img alt="Card image cap" class="card-img-top img-responsive" src="guide-icons/servers.png" style="margin-left: auto;margin-right: auto;width: 30%;"/>
+		<h5 class="card-title text-center">New Upgraded HPC Guides</h5>
```

Comment:
```suggestion
		<h5 class="card-title text-center">Guides for New Cluster Configuration (Sept 17)</h5>
```

Reply:
not sure how many characters will fit in the box without getting squashed (not sure if it will wrap)